pkgs/development/tools/analysis/cargo-tarpaulin/default.nix

```
@@ -26,5 +26,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/xd009642/tarpaulin";
     license = with licenses; [ mit /* or */ asl20 ];
     maintainers = with maintainers; [ hugoreeves ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" ];
```

Sounds good, what about this one (Does not really make a big difference).

```suggestion
    platforms = lib.platforms.x86_64;
```

---

I did not mark this as any x86_64 because officially cargo-tarpaulin only supports x86_64-linux [[1](https://github.com/xd009642/tarpaulin/blob/3a241f57c0efb7b03cc76b73cf32d1c5bd3bbacb/README.md?plain=1#L17)], it will likely not build/run on other x86_64 OSs out of the box.

I'm not sure what other x86_64 OSs people would be using with Nix (BSD?), but either way is fine by me.